cmd/expired-authz-purger/main.go

```diff
@@ -71,33 +91,30 @@ func (p *expiredAuthzPurger) purgeAuthzs(purgeBefore time.Time, yes bool) (int64
 		}
 	}
 
-	rowsAffected := int64(0)
-	for {
-		result, err := p.db.Exec(`
-			DELETE FROM pendingAuthorizations
-			WHERE expires <= ?
-			LIMIT ?
-			`,
-			purgeBefore,
-			p.batchSize,
-		)
+	for _, id := range ids {
+		// Delete challenges + authorization
+		_, err := p.db.Exec("DELETE FROM challenges WHERE authorizationID = ?", id)
 		if err != nil {
-			return rowsAffected, err
+			return err
 		}
-		rows, err := result.RowsAffected()
+		_, err = p.db.Exec(fmt.Sprintf("DELETE FROM %s WHERE id = ?", table), id)
 		if err != nil {
-			return rowsAffected, err
+			return err
 		}
+	}
 
-		p.stats.Inc("PendingAuthzDeleted", rows)
-		rowsAffected += rows
-		p.log.Info(fmt.Sprintf("Progress: Deleted %d (%d total) expired pending authorizations", rows, rowsAffected))
+	p.log.Info(fmt.Sprintf("Deleted a total of %d expired authorizations from %s", len(ids), table))
+	return nil
+}
 
-		if rows < p.batchSize {
-			p.log.Info(fmt.Sprintf("Deleted a total of %d expired pending authorizations", rowsAffected))
-			return rowsAffected, nil
+func (p *expiredAuthzPurger) purgeAuthzs(purgeBefore time.Time, yes bool) error {
+	for _, table := range []string{"pendingAuthorizations", "authz"} {
```

Comment:
I notice that `sa/authz.go` has a [`authorizationsTables` var](https://github.com/letsencrypt/boulder/blob/e2b2511898f5d76ea68e26402521d628751d97f4/sa/authz.go#L13:L16) that serves the same purpose as this `[]string{}` constant. We also duplicate it [in the SA's `RevokeAuthorizationsByDomain()` func](https://github.com/letsencrypt/boulder/blob/e2b2511898f5d76ea68e26402521d628751d97f4/sa/sa.go#L725). Do you think it's worth exporting this from one place and updating the other places to reference it? I feel like the duplication is starting to add up. (Maybe this is better suited for a follow-up PR? Thoughts?)